frontend/src/individual_edit.tsx

```diff
@@ -241,7 +248,7 @@ export function IndividualEdit({id}: {id: string | undefined}) {
                 onChange={(event) => {updateField('number', event.currentTarget.value)}}
               />
               <TextField
-                disabled={!user?.canEdit(individual.genebank)}
+                disabled={!canManage}
```

Comment:
No need to check `isNew` here?

Reply:
Yes a herd owner might want to input a new rabbit In the system that use a old system aka a paper certificate and if so they would like to input the certificate number from the physical paper I'm the system.
If the the herd owner want to issue a digital certificate of the New rabbit the certificate number should come from our system. So possible three different number series of the certificate number.

Reply:
ok! :) I had understood things as the certificate always coming from the manager, but that's an easy fix!